.markdownlint.yml

```diff
@@ -0,0 +1,10 @@
+"default": true
+"MD002": false # First heading should be a top level heading
+"MD007": # Unordered list indentation
+  indent: 4
+"MD009": false # Trailing spaces
+"MD013": false # Line length
+"MD025": false # Multiple top level headings in the same document
+"MD026": false # Trailing punctuation in heading
+"MD033": false # Inline HTML
+"MD041": false # Inline HTML
```

Comment:
Also, what is the point of turning all these off?  Will we have checks if they are all disabled?  Or are there more?

Reply:
There is a big list of rules: https://github.com/DavidAnson/markdownlint#rules--aliases. These are just some the most noisy once. Might be a good thing to reenable them one by one as a follow up.

Reply:
Ah okay, works for me then.